docs/setting-up/server/docker.md

```diff
@@ -101,6 +101,9 @@ How to run PMM Server with Docker based on our [Docker image].
 
 ---
 
+!!! caution alert alert-warning "Important"
```

Comment:
The backup is just for PMM, right? No databases are affected?

Reply:
Yes @ggaspard